daemon/database/artistrepo.py

```diff
@@ -1,16 +1,27 @@
 import logging
 
 from decorator.injectdbsession import inject_db_session
-
 from .entity import Artist
+from sqlalchemy.orm import lazyload
 
 
 @inject_db_session()
 class ArtistRepo:
     logger = logging.getLogger(__name__)
 
-    def getArtistsByName(self, artist_name):
-        query = self._session.query(Artist).filter(Artist.name == artist_name)
+    def get_artists_by_page(self, index, offset):
+        query = self._session.query(Artist).options(lazyload("albums")).order_by(
+            Artist.name).limit(offset).offset((index - 1) * offset)
+        return query.all()
+
+    def get_artists_list(self):
+        query = self._session.query(Artist).options(
+            lazyload("albums")).order_by(Artist.name)
+        return query.all()
+
+    def get_artist(self, artist_name):
```

Comment:
This function should be named `get_artist_by_name` since we may have other ways to get the artist, such as `source id`.